docs/configuration.rst

```diff
@@ -33,7 +33,7 @@ below:
    * - ``DD_TAGS``
      - String
      -
-     - Set global tags to be attached to every span. e.g. ``key1:value1,key2,value2``. Added in ``v0.38.0``.
+     - Set global tags to be attached to every span. Value must be either comma or space separated. e.g. ``key1:value1,key2,value2`` or ``key1:value key2:value2``. Added in ``v0.38.0``.
```

Comment:
I'd actually not update this, keep all of our docs as comma separated to try and keep consistency with usage.

What do you think?

Reply:
Problem is that `DD_TAGS` is specified in the agent docs as space separated:

https://docs.datadoghq.com/agent/docker/?tab=standard#global-options

Reply:
If we document support for space as separator, then we should be careful with the indicated version. Whilst this was added in 0.38.0, support for spaces comes with a future release